x-pack/plugins/observability_solution/apm/server/routes/traces/get_trace_items.ts

```diff
@@ -22,6 +22,7 @@ import {
   ERROR_LOG_MESSAGE,
   EVENT_OUTCOME,
   FAAS_COLDSTART,
+  FALLBACK_TIMESTAMP,
```

Comment:
Same in here
`TIMESTAMP` & `TIMESTAMP_US`, it's more clear IMHO

Reply:
fixed